packages/sparrowql-graphql/src/graph.ts

```diff
@@ -162,7 +194,7 @@ export function astToPipeline(info: GraphQLResolveInfo, options: Options) {
             position[part] = {};
           }
 
-          position = position[part] as Record<string, unknown>;
+          position = position[part];
         }
 
         position[path[0]] = `$${target}`;
```

Comment:
There is type mismatch probably due to other wrong typings. Is this correct? @radekmie 

```ts
  export type ProjectionType = Record<string, string>;
  const inflateMap: Record<string, ProjectionType | string> = {};
```

https://github.com/vazco/sparrowql/blob/a8b84ceecc67026cc5d8ef0b8f9f129d238aa9ec/packages/sparrowql-graphql/src/graph.ts#L159

Reply:
It's fine, but it's more like:
```ts
type InflateMap = string | Record<string, InflateMap>;
```